run.sh

```diff
@@ -0,0 +1,7 @@
+# cd pipeline_dir
+# docker pull phenoscape/pipeline-tools:latest
+
+docker run -v "$(pwd)":/pipeline -ti phenoscape/pipeline-tools
```

Comment:
This won't work. You'll have to have to default executable in the Dockerfile, or you must provide the program to run here. I.e., you can't "run" an image, there has to be program inside of the container that is to be run.

Reply:
This was launching the bash shell in the container with the requisite environment, which I have been using to manually run the relevant `make` command. But yeah, I have now changed it to accept the command from the user on the command line.

Reply:
Even if the intent is to run `bash`, it's poor practice to assume that that's what will happen. (If the Dockerfile changes, this assumption may quickly go to hell.)